test_runner/random_test_runner/Readme.md

```diff
@@ -156,7 +156,7 @@ ros2 run lanelet2_scene_builder lanelet2_scene_builder --ros-args -p osm_file:=<
 ```
 7. Execute `random_test_runner` launch with `spawn_ego` parameter and correct map name:
```

Comment:
`spawn_ego` -> `simulator_type` here as well. 

Overall thanks a lot for this PR!